docs/reference/commandline/swarm_join.md

```diff
@@ -17,52 +17,52 @@ Usage:  docker swarm join [OPTIONS] HOST:PORT
 Join a Swarm as a node and/or manager
 
 Options:
-      --ca-hash string      Hash of the Root Certificate Authority certificate used for trusted join
       --help                Print usage
       --listen-addr value   Listen address (default 0.0.0.0:2377)
-      --manager             Try joining as a manager.
-      --secret string       Secret for node acceptance
+      --token string        Token for entry into the swarm
 ```
 
-Join a node to a Swarm cluster. If the `--manager` flag is specified, the docker engine
-targeted by this command becomes a `manager`. If it is not specified, it becomes a `worker`.
+Join a node to a Swarm cluster. If the token given to the `--token` flag is a manager token,
```

Comment:
@tiborvass , @aaronlehmann , either way, we'll need to review the doc set for consistent usage in addition to this PR. So I've opened a docs issue: https://github.com/docker/docker/issues/24905